src/charts/row-chart.js

```diff
@@ -411,8 +411,21 @@ export default function rowChart (parent, chartGroup) {
                   const thisLabel = d3.select(this)
 
                   const width = Math.abs(rootValue() - _x(_chart.valueAccessor()(d)))
-                  const measureWidth = thisLabel.node().getBBox().width
-                  const dimWidth = _chart.svg().select("text.value-dim._" + i).node().getBBox().width
+                  
+                  //
+                  // handle Firefox getBBox bug
```

Comment:
This comment maybe isn't so helpful, it certainly won't be in a few months. Maybe link to the "bug"

Reply:
good call